0-gioi-thieu-tong-quan/0-6-huong-dan-cai-dat-codeblocks-linux/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Hướng dẫn cài đặt Code::blocks môi trường phát triển ứng dụng (Ubuntu)
+---
+
+Tiếp the trong chủ đề này, chúng ta sẽ cài đặt Code::block thiết lập môi
+trường lập trình ngôn ngữ C/C++ trên Ubuntu nhằm phục vụ việc học tập của các
+bạn chỉ  có máy Linux mà cụ thể ở đây là Ubuntu.
+
+## Giới thiệu Code::blocks
+
+Code::Blocks là một Integrated Development Environment (IDE) tức là một môi
+trường tích hợp bao gồm nhiều công cụ khác nhau như chương trình viết mã
+lệnh hay code editor, chương trình sửa lỗi hay debugger, chương trình mô phỏng
+ứng dụng khi chạy thực tế hay simulator... hỗ trợ việc lập trình C/C++ trên
+nền tảng Linux, Những tính năng nổi bậc phải kể đến như:
+
+  * Các bộ compilers miễn phí
+  * Source formatter (dựa trên AStyle)
+  * Tạo C++ classes theo từng bước
+  * Code-completion / symbols-browser (Gợi ý Coding)
+  * Default MIME handler
+  * Tạo Code::Blocks plugins dễ dàng
+  * To-do list
+  * Autosave (cái này hữu ích với nhiều trường hợp nhất định)
+  * Rất nhẹ
+  * Code::blocks miễn phí
+
+Và còn nhiều nữa, các bạn tự khám phá nhé.
+
+> **Chú ý:** Ngoài Code::Blocks ra còn có các IDE khác hỗ trợ lập trình C/C++
+> trên Linux như Eclipse, QtCreator... hoặc chỉ cần lệnh bạn cũng có thể compile
+> C/C++. Bạn có thể tìm hiểu thêm trên mạng.
+
+## Cách cài đặt Code::blocks
+
+Chúng ta có thể cài đặt Code::block bằng hai cách sau
+
+### 1. Qua cửa sổ lệnh
+
+Trong cách này, dành cho các bạn đã quá quen với cửa sổ lệnh trong linux,
+các bước vô cùng đơn giản, truy cập vào Terminal bằng cách dùng tổ hợp phím
+`Ctrl` + `Alt` + `T` và nhập vào lần lượt các lệnh sau:
+
+```bash
```

Comment:
Branch của em up-to-date với `daynhauhoc:master` rồi mà.

Nếu Github có nút <kbd>Update branch</kbd> thì cứ bấm vào

Nếu không có (tức là có conflict) thì em đọc hướng dẫn ở đây. Resolve conflict trong khi rebase 

https://help.github.com/articles/syncing-a-fork/